bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/DateTextField.java

```diff
@@ -130,7 +242,16 @@ public void renderHead(final IHeaderResponse response) {
      * @return initializer script
      */
     protected CharSequence createScript(final DateTextFieldConfig config) {
-        return $(this).chain("datepicker", config).get();
+        CharSequence script = $(this).chain("datepicker", config).get();
+        if(eventMap.isEmpty()) {
+            return script;
+        }
+        StringBuilder sb = new StringBuilder(script.subSequence(0, script.length()-1));
```

Comment:
I see. It removes the trailing semicolon.
I think jquery-selectors API provides better way of adding `.on()`. We just have to postpone calling `.get()`.